Documentation/DesignGuidelines.md

```diff
@@ -10,7 +10,7 @@ resource for getting up to speed on the main types and concepts provided by RAC.
 
 **[The `Event` contract](#the-event-contract)**
 
- 1. [`Next`s provide values or indicate the occurrence of events](#nexts-provide-values-or-indicate-the-occurrence-of-events)
+ 1. [`value`s provide values or indicate the occurrence of events](#nexts-provide-values-or-indicate-the-occurrence-of-events)
```

Comment:
Should the headers be more consistent? Perhaps the three below should use backticks too.